configure.ac

```diff
@@ -2656,6 +2656,9 @@ yes)
 esac
 
 case "$CC" in
+*mpicc*)
+    CFLAGS_NODIST="$CFLAGS_NODIST"
```

Comment:
Strictly, this line is not needed; we could for example do:

```suggestion
```

I'm also fine with the current line.